src/sentry/migrations/0146_backfill_members_alert_write.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-07-28 16:24
+from __future__ import unicode_literals
+
+from django.db import migrations, transaction
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+import logging
+
+ORGANIZATION_OPTION_KEY = 'sentry:alerts_member_write'
+
+
+def backfill_existing_orgs(apps, schema_editor):
+    """
+    Backfill the OrganizationOption alerts_member_write to be False for existing orgs
+    """
+    Organization = apps.get_model("sentry", "Organization")
+    OrganizationOption = apps.get_model("sentry", "OrganizationOption")
+
+    for org in RangeQuerySetWrapperWithProgressBar(Organization.objects.filter(status=0)):
```

Comment:
There's no index on `status=0`, so I'd suggest just iterating over `Organization.objects.all()` instead, and add a:

```python
if org.status != 0:
    continue
```

Reply:
Yeah, just to add to this:

```
getsentry=# select count(*), status from sentry_organization group by status order by 1 desc;
 count  | status
--------+--------
 412569 |      0
   1317 |      2
    514 |      1
(3 rows)
```

By far, most of them are 0, so this filter will just be more efficient to do in Python and easier on our database.